examples/ksml-runner.yaml

```diff
@@ -1,10 +1,72 @@
 ksml:
-  # The examples directory is mounted to /ksml in the run script
-#  configDirectory: /ksml # When not set defaults to the working directory
-#  schemaDirectory: /ksml # When not set defaults to the config directory
-#  storageDirectory: /tmp # When not set defaults to the default JVM temp directory
+  # The examples directory is mounted to /ksml in the Docker container
+  configDirectory: .              # When not set defaults to the working directory
+  schemaDirectory: .              # When not set defaults to the config directory
+  storageDirectory: /tmp          # When not set defaults to the default JVM temp directory
+
+  # This section defines if a REST endpoint is opened on the KSML runner, through which
+  # state stores and/or readiness / liveness probes can be accessed.
+  applicationServer:
+    enabled: false                # Set to true to enable, or false to disable
+    host: 0.0.0.0                 # IP address to bind the REST server to
+    port: 8080                    # Port number to listen on
+
+  # This section defines whether a Prometheus endpoint is opened to allow metric scraping.
   prometheus:
-  enablePipelines: true
+    enabled: true                 # Set to true to enable, or false to disable
+    host: 0.0.0.0                 # IP address to bind the Prometheus agent server to
+    port: 8080                    # Port number to listen on
```

Comment:
This is the same port as the Application Server, and should not work. Default port should is 9999